angrmanagement/ui/workspace.py

```diff
@@ -143,6 +144,29 @@ def on_function_tagged(self):
     # Public methods
     #
 
+    def new_disassembly_view(self):
+        """
+        Add a new disassembly view into the
+        central_widget with a unique id
+
+        :return:    None
+        """
+
+        dis_views = self.view_manager.views_by_category['disassembly']
+        dis_ids = [int(view.caption.split("-")[-1]) for view in dis_views]
```

Comment:
Instead of parsing the name of the view, can we add an `index` property to `BaseView`, then make `BaseView.caption` a property that always derives from `BaseView.base_caption` and `BaseView.index`?

Reply:
no problem, will add that now